_data/simple-icons.json

```diff
@@ -12515,7 +12515,7 @@
         },
         {
             "title": "Talenthouse",
-            "hex": "FFFFFF",
+            "hex": "000000",
             "source": "https://www.talenthouse.com"
```

Comment:
Should plan to remove this one maybe?

Reply:
np. Let's do this in another pull request. On this pull request, I just want to drop all `#FFFFFFF` color.